platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/GlSurfaceViewRenderThread.java

```diff
@@ -0,0 +1,25 @@
+package com.mapbox.mapboxsdk.maps.renderer;
+
+import android.opengl.GLSurfaceView;
+
+/**
+ * {@link RenderThread} implementation that schedules using the
+ * {@link GLSurfaceView} thread.
+ */
+public class GlSurfaceViewRenderThread implements RenderThread {
```

Comment:
Why so much indirection here? Can `NativeMapView` just have a `GLSurfaceView` reference?

Reply:
This is in preparation of bringing back the TextureView (with a separate rendering thread).